src/net_processing.cpp

```diff
@@ -2777,11 +2787,28 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         std::vector<CAddress> vAddrOk;
         int64_t nNow = GetAdjustedTime();
         int64_t nSince = nNow - 10 * 60;
+
+        // Update/increment addr rate limiting bucket.
+        const auto current_time = GetTime<std::chrono::microseconds>();
+        if (peer->m_addr_token_bucket < MAX_ADDR_TO_SEND) {
```

Comment:
This `MAX_ADDR_TO_SEND` is semantically the bucket size, so if a new constant is added it should be used here.

Reply:
Done.